.travis.yml

```diff
@@ -24,3 +24,4 @@ before_script:
 
 script:
   - './node_modules/.bin/jest --maxWorkers=2 --ci'
+  - BRANCH="${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}" && yarn canarist -r [https://github.com/xing/hops.git -b $BRANCH -c] -r [https://github.com/foosbyte/kicker-trainer.git -c 'yarn build -p' -c 'yarn test'] -m '{"resolutions":{"typescript":"3.2.4"}}'
```

Comment:
I'd like to have this command separated from the travis-related branch fiddling so we're able to run it locally without copy&paste hassle.